Doxyfile

```diff
@@ -44,7 +44,7 @@ PROJECT_NUMBER         = @VERSION@
 # for a project that appears at the top of each page and should give viewer a
 # quick idea about the purpose of the project. Keep the description short.
 
-PROJECT_BRIEF          = "@CPACK_PACKAGE_DESCRIPTION_SUMMARY@"
+PROJECT_BRIEF          = "@DOXYGEN_PROJECT_BRIEF@"
```

Comment:
I updated the documentation (I had forgot to mention the new DOXYGEN_PROJECT_BRIEF).

If I look at Equalizer for instance, the CPACK_PACKAGE_DESCRIPTION_SUMMARY is "Parallel Rendering Framework" which is a shorter version of EQUALIZER_DESCRIPTION (which spans two lines). But EQUALIZER_DESCRIPTION is then completely unused..! (except, already in DoxygenRule where I used it for the metadata for the Jekyll-generated index.html).
I'm sure this change is the correct thing to do - EQUALIZER_DESCRIPTION will now be used by default by doxygen, Jekyll and CPack, and each one can still be overwritten for specific use cases.

For projects which currently have two descriptions, the decision to keep one or both is project specific. I am not sure for instance which one you want to keep for Equalizer. But can we already merge this and slowly adapt projects along the way?